### DISCUSSION TITLE
Is it possible to merge the chunks of a paragraph

### DISCUSSION THREAD

**Discussion by Polperobis** [davidgohel/officer#526] [Q&A]
I’m using body_replace_all_text(). It works fine with a simple docx created by copy/paste but fail to find the text to replace in my docx. Using docx_show_chunk() I have seen that in my docx the paragraphs are made of plenty of chunks (with a lot of /n). But I don't know how to solve the problem… Would it be a good idea to start by merging the chunks in the same paragraph? If so, how can this merging be carried out?

**Comment by davidgohel** [comment]
sorry, it's not possible. As documented in the manual, best solution is to copy paste pure text to avoid multiple chunk. My opinion is that this is not a good option and I recommand to use Word computed fields and package 'doconv', see https://github.com/ardata-fr/doconv#update-word-fields-and-toc
